tests/scripts/deploy-chartmuseum

```diff
@@ -42,8 +46,8 @@ helm plugin install https://github.com/chartmuseum/helm-push.git
 helm repo add chartmuseum http://localhost:8080
 
 # Download needed helm charts
-helm pull oci://registry.opensuse.org/isv/rancher/elemental/dev/charts/rancher/elemental-operator-crds-chart
-helm pull oci://registry.opensuse.org/isv/rancher/elemental/dev/charts/rancher/elemental-operator-chart
+helm pull oci://registry.opensuse.org/isv/rancher/elemental/$OPERATOR_VERSION/charts/rancher/elemental-operator-crds-chart
+helm pull oci://registry.opensuse.org/isv/rancher/elemental/$OPERATOR_VERSION/charts/rancher/elemental-operator-chart
```

Comment:
Will it be possible to set this ${OPERATOR_VERSION} from operator_repo input value? I am mostly thinking what would happen if we set operator_repo to something like `oci://registry.opensuse.org/isv/rancher/elemental/mybranch/charts/rancher`, so not `staging` or `dev`.